docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -387,6 +387,7 @@ For the `s3` backend, the following additional properties are supported in the `
 - `skip_bucket_accesslogging`: _DEPRECATED_ If provided, will be ignored. A log warning will be issued in the console output to notify the user.
 - `skip_bucket_root_access`: When `true`, the S3 bucket that is created will not be configured with bucket policies that allow access to the root AWS user.
 - `skip_bucket_enforced_tls`: When `true`, the S3 bucket that is created will not be configured with a bucket policy that enforces access to the bucket via a TLS connection.
+- `skip_bucket_public_access_blocking`: When `true`, the S3 bucket that is created will not have public access blocking enabled.
```

Comment:
Not sure if this might need a "why", as it might not be immediately clear what this is useful for?

Reply:
I think this is poorly named. It's hard to understand. It would be clearer to call this "disable_public_access_blocking" or "enable_public_access". So when this variable is `true`, public access is allowed.